src/harmony/Radio/Radio.module.css

```diff
@@ -31,9 +29,11 @@
 
     outline: none;
     padding: 0;
-    margin: 0.125rem; /* with this prop getting same size as Checkbox */
+    margin: 0.03125em; /* with this prop getting same size as Checkbox */
```

Comment:
magic

Reply:
`<input type="radio" />`  has `15px` dimensions by sides
`<input type="checkbox" />`  has `16px` dimensions by sides

This magic number needs for correctly positioning inputs on the middle of the first line